src/gConsent/manage/getAccessGrant.test.ts

```diff
@@ -20,17 +20,30 @@
 //
 
 import { jest, it, describe, expect } from "@jest/globals";
+import { Response } from "cross-fetch";
+import type * as CrossFetch from "cross-fetch";
+
 import { mockAccessApiEndpoint } from "../request/request.mock";
 import { mockAccessGrantVc, mockConsentRequestVc } from "../util/access.mock";
 import { getAccessGrant } from "./getAccessGrant";
 
-jest.mock("cross-fetch");
+jest.mock("cross-fetch", () => {
+  const crossFetch = jest.requireActual("cross-fetch") as jest.Mocked<
+    typeof CrossFetch
+  >;
+  return {
+    // Do no mock the globals such as Response.
+    ...crossFetch,
+    Response: crossFetch.Response,
```

Comment:
```suggestion
```

this is handled by the line above, or should be.

Reply:
Oops yeah looks like a leftover of a test 03265e0161d371884234c0951c685076bb738fc2